docs/guides/migrating-from-cawemo.md

```diff
@@ -0,0 +1,56 @@
+---
+id: migrating-from-cawemo
+title: Migrating diagrams from Cawemo
+description: "Migrating diagrams from Cawemo to Camunda Platform 8 Web Modeler"
+---
+
+This guide steps through how to migrate BPMN diagrams created in Cawemo to Camunda Platform 8 Web Modeler.
```

Comment:
I would link to Web Modeler here on first reference.

Reply:
check